csharp/Build/Facebook.Yoga.NativeInterop.targets

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
This target is used by the test programs to ensure that the proper Native DLL is copied to current PlatformTarget output path... Normally would be done by a Nuget package, however, since the C# wrapper, native DLL, and test project are all in the same solution we include this target in the test project's csproj.